lib/bank_account.rb

```diff
@@ -56,26 +56,37 @@ def self.find(id)
     end
 
     def self.associate_everything
-      account_and_owner_csv = CSV.read("support/account_owners.csv")
       everything_array = []
-      account_and_owner_csv.each do |line|
+      CSV.foreach("support/account_owners.csv") do |line|
         each_account = Bank::Account.find(line[0].to_i)
         each_owner = Bank::Owner.find(line[1].to_i)
-        each_account.add_owner(each_owner)
+        each_account.owner = each_owner
```

Comment:
Since you switched to setting the owner directly, do you still need the `add_owner` method?